widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -10,6 +10,16 @@ export const Container = styled('div', {
   width: '95vw',
   maxWidth: '390px',
   maxHeight: '700px',
+  fontFamily: '$widget',
+  boxSizing: 'border-box',
+  '& *, *::before, *::after': {
```

Comment:
I think this global style shouldn't be here because it's not directly related to `Layout` and affect the whole app. You can define a new global style besides `globalFont`.

Reply:
I also don't know why you are `inherit` the box sizing, setting `border-box` would be more appropriate because by adding a `box-sizing` in somewhere in DOM, it will change the behavipor of its children.

Reply:
`listStyleType` is also for lists, not all the elements.